libs/langgraph/src/pregel/index.ts

```diff
@@ -1645,10 +1644,31 @@ export class Pregel<
    * @param options - Configuration options for streaming
    * @returns An async iterable stream of graph state updates
    */
-  override async stream(
+  // @ts-expect-error Return type of `stream()` differs from `invoke()`, which is expected.
```

Comment:
Unsure if there are downsides to this, but feels like a good shortcut to make, since I don't want to wait for a @langchain/core update to have a different generic parameter just for `stream`. Open to suggestions.